cddl/ar4si.cddl

```diff
@@ -0,0 +1,31 @@
+$.start.$ = trustworthiness-vector / trust-tier / verifier-id
```

Comment:
Should  `trustworthiness-claim` be here as well?

Reply:
yes.  However, this needs fixing in [upstream AR4SI](https://github.com/ietf-rats-wg/draft-ietf-rats-ar4si/blob/main/cddl/start.cddl) and then re-import.

Reply:
see https://github.com/ietf-rats-wg/draft-ietf-rats-ar4si/pull/32

Reply:
Addressed in [c454333](https://github.com/thomas-fossati/draft-ear/pull/44/commits/c454333b27d927cb81b7f6868f97cb30cd01e991)